### PR TITLE
C#: Add Vector2/3/4i.MAX and MIN

### PR DESCRIPTION
**Pull request by Chaosus**
Continuation of https://github.com/godotengine/godot/pull/81741 (for C#).

### CODE REVIEW

File: modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4I.cs [hunk 1]
```diff
@@ -228,9 +228,23 @@ readonly get
...
 
         // Constants
+        private static readonly Vector4I _min = new Vector4I(int.MinValue, int.MinValue, int.MinValue, int.MinValue);
+        private static readonly Vector4I _max = new Vector4I(int.MaxValue, int.MaxValue, int.MaxValue, int.MaxValue);
+
         private static readonly Vector4I _zero = new Vector4I(0, 0, 0, 0);
         private static readonly Vector4I _one = new Vector4I(1, 1, 1, 1);
 
+        /// <summary>
+        /// Min vector, a vector with all components equal to <see cref="Int32.MinValue"/>. Can be used as a negative integer equivalent of <see cref="Vector4.Inf"/>.
```
Comment (raulsntos) [inline modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4I.cs:238]: nit: To be consistent with the rest of the documentation.

```suggestion
        /// Min vector, a vector with all components equal to <see cref="int.MinValue"/>. Can be used as a negative integer equivalent of <see cref="Vector4.Inf"/>.
```

File: modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4I.cs [hunk 2]
```diff
@@ -228,9 +228,23 @@ readonly get
...
+        /// </summary>
+        /// <value>Equivalent to <c>new Vector4I(int.MinValue, int.MinValue, int.MinValue, int.MinValue)</c>.</value>
+        public static Vector4I Min { get { return _min; } }
+        /// <summary>
+        /// Max vector, a vector with all components equal to <see cref="Int32.MaxValue"/>. Can be used as an integer equivalent of <see cref="Vector4.Inf"/>.
```
Comment (raulsntos) [inline modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4I.cs:243]:
```suggestion
        /// Max vector, a vector with all components equal to <see cref="int.MaxValue"/>. Can be used as an integer equivalent of <see cref="Vector4.Inf"/>.
```

**Review by raulsntos — APPROVED** [review]
Just a nit with the documentation, but otherwise looks great. Thanks!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**